test/SaveHandler/CacheTest.php

```diff
@@ -103,4 +103,17 @@ public function testReadWriteTwice()
 
         $this->assertEquals($this->testArray, unserialize($saveHandler->read($id)));
     }
+
+    public function testReadShouldAlwaysReturnString()
+    {
+        $cacheStorage = $this->prophesize('Zend\Cache\Storage\StorageInterface');
+        $cacheStorage->getItem('242')->willReturn(null);
+        $this->usedSaveHandlers[] = $saveHandler = new Cache($cacheStorage->reveal());
+
+        $id = '242';
```

Comment:
yes, but the tests is not proving the (string) cast then, as `242` must be string, and 242 is not string.